build.gradle.kts

```diff
@@ -20,11 +20,11 @@ val kafkaEmbeddedVersion = "2.4.0"
 val kithHodemeldingVersion = "2019.07.30-12-26-5c924ef4f04022bbb850aaf299eb8e4464c1ca6a"
 val kithApprecVersion = "2019.07.30-04-23-2a0d1388209441ec05d2e92a821eed4f796a3ae2"
 val kluentVersion = "1.51"
-val ktorVersion = "1.2.6"
+val ktorVersion = "1.3.1"
```

Comment:
Hvis det blir kranglete med denne versjonen kan du dra inn nyere versjon av kun diagnosekodene, ikke hele syfosmCommon. Jeg måtte gjøre det for sykmeldinger-backend, for der kan vi ikke ha enkeltrequests som stadig feiler.